src/js/utils/specs/date-utils-spec.js

```diff
@@ -0,0 +1,145 @@
+define(
+    [
+        '../../utils/spec-helpers/spec-helpers.js',
+        '../date-utils.js',
+        'moment',
+        'moment-timezone'
+    ],
+    function(
+        SpecHelpers,
+        DateUtils,
+        moment,
+        momentTZ
+    ) {
+        'use strict';
+
+        describe('DateUtils', function() {
+            beforeEach(function() {
+                setFixtures('<div class="test"></div>');
+            });
+
+            describe('stringToMoment', function() {
+                it('returns an empty string if provided no time', function() {
```

Comment:
See above comment about returning `undefined` instead since the function is expected to return an object.